### PR TITLE
Added demo example for reactive forms

### LINKED ISSUE #97: CKEditor no longer showing data from formControlName
Before i updated to the 1.1.0 it was possible to pass the form field data via formControlName to the CKEditor.
Today i updated to this new version and all editors are no longer taking data from formControlName (you cannot see it in the editor).
They only pass the data back to the formControl.

Used Angular Version: 7.2.14
Last working version for me: 1.0.1

### PR DESCRIPTION
**Pull request by dkrahn**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Example: Added a demo using reactive forms(formControlName). Closes #97.

---

### Additional information
Based on the form example. Test where copied from there too.

### CODE REVIEW

File: src/app/demo-reactive-form/demo-reactive-form.component.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import {
+	Component,
+	ViewChild,
```
Comment (ma2ciek) [inline src/app/demo-reactive-form/demo-reactive-form.component.ts]: It isn't used anywhere.

File: src/app/demo-reactive-form/demo-reactive-form.component.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+@Component({
+	selector: 'app-demo-reactive-form',
+	templateUrl: './demo-reactive-form.component.html',
+	styleUrls: ['./demo-reactive-form.component.css']
+})
+export class DemoReactiveFormComponent implements AfterViewInit {
+
+	public Editor = ClassicEditorBuild;
+
+	demoReactiveForm = new FormGroup({
```
Comment (ma2ciek) [inline src/app/demo-reactive-form/demo-reactive-form.component.ts]:
```suggestion
	public demoReactiveForm = new FormGroup({
```

File: src/app/demo-reactive-form/demo-reactive-form.component.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<h2>Integration with forms (<code>ngModel</code>)</h2>
```
Comment (ma2ciek) [inline src/app/demo-reactive-form/demo-reactive-form.component.html]:
```suggestion
<h2>Integration with reactive forms (<code>ngModel</code>)</h2>
```

**Comment by ma2ciek** [comment]
Thanks for the PR!

I'm sorry, I hadn't time to look at it earlier. I left a few comments.

File: src/app/demo-reactive-form/demo-reactive-form.component.ts (hunk 1 above)
Comment (dkrahn) [reply src/app/demo-reactive-form/demo-reactive-form.component.ts]: Removed. Thanks

**Comment by ma2ciek** [comment]
Thanks again @dkrahn for the PR! I'm merging it.

**Pull request merged**